lib/solidus_dev_support/rake_tasks.rb

```diff
@@ -34,10 +34,18 @@ def install_test_app_task
       ::CLOBBER.include test_app_path
 
       namespace :extension do
+        # We need to go back to the gem root since the upstream
```

Comment:
I think this comment should go in a commit message, or at the very least above `cd root` rather than here.

Reply:
That was my first instinct as well, but the whole task is there just to add that, so I figured it was all-or-nothing, let me know if you still prefer to have it down there.

Reply:
Let's keep it here, but can you remove the todo and just open an issue on Solidus?